lineapy/db/relational.py

```diff
@@ -337,6 +338,22 @@ class GlobalNodeORM(BaseNodeORM):
     }
 
 
+class IfElseNodeORM(BaseNodeORM):
+    __tablename__ = "if_else_node"
+
+    id = Column(String, ForeignKey("node.id"), primary_key=True)
+    test_id = Column(String, ForeignKey("node.id"))
+    unexec_contents = Column(String, ForeignKey("node.id"))
```

Comment:
optional but you can tie this to literalnodeorm.id. this makes it restrictive and helps validate more strongly. same goes for others but again this might be needless.